Clinical-Genomics-Lund/gens
# Added javascript unit tests and code linting

This PR adds the basic infrastructure to lint and test gens javasscript module using [jest](https://jestjs.io/) and [eslint](https://eslint.org/). Javascript code as reorganized into a module to enhance the linting and testing and new commands was added to npm.

**How to test**:
1. checkout the branch
2. install node packages with `npm install -D`
3. run `npm run lint` 
4. start gulp server `npm run watch`
5. made some addition to the new entrypoint in `asasets/js/gens.js` such as `alert("python is king")`
6. reload Gens in browser and verity that the code has been updated
7. test building production assets with `npm run build` and repeat step 6
8. run unit tests with `npm run test` and verify that things are working correctly

**Review:**
